src/transform/bpcells_regress_out/script.R

```diff
@@ -0,0 +1,43 @@
+cat("Loading libraries\n")
+library(glue)
+library(BPCells)
+requireNamespace("anndata", quietly = TRUE)
+requireNamespace("reticulate", quietlye = TRUE)
+mudata <- reticulate::import("mudata")
+
+## VIASH START
+par <- list(
+    input = "resources_test/pbmc_1k_protein_v3/pbmc_1k_protein_v3_filtered_feature_bc_matrix.h5mu",
+    output = "output.h5mu",
+    modality = "rna"
+)
+## VIASH END
+
+# Read the h5mu file and make var names unique
+mdata <- mudata$read_h5mu(par$input)
+mdata$var_names_make_unique()
+
+# Fetch modality AnnData and convert to an iterable matrix
+adata <- mdata$mod[[par$modality]]
+mat <- as.matrix(adata)
```

Comment:
We need to avoid calling `as.matrix` because this will convert a sparse matrix into a dense one.